packages/react-router-native/docs/guides/animation.md

```diff
@@ -109,117 +110,117 @@ class Parent extends Component {
     return (
       <View>
         <Switch>
-          <Route path="/settings"/>
-          <Route path="/notifications"/>
+          <Route path="/settings" />
+          <Route path="/notifications" />
         </Switch>
       </View>
-    )
+    );
   }
 }
 ```
 
 Once that works without animations, we're ready to add an animation around it.
 
-
 ```jsx
 <AnimatedChild
   anim={this.state.anim}
   atParent={this.props.match.isExact}
   animating={this.state.animating}
 >
   <Switch location={this.props.location}>
-    <Route path="/settings"/>
-    <Route path="/notifications"/>
+    <Route path="/settings" />
+    <Route path="/notifications" />
   </Switch>
 </AnimatedChild>
 ```
 
-It's important to use a [`<Switch>`][Switch]. It will ensure that only one route can match, and therefore gives us a single element on `props.children` to hang on to and render during the animation. Finally, you *must* pass the location to `Switch`. It prefers `props.location` over the internal router location, which enables the saved child element to be renered later and continue to match the old location.
+It's important to use a [`<Switch>`][switch]. It will ensure that only one route can match, and therefore gives us a single element on `props.children` to hang on to and render during the animation. Finally, you _must_ pass the location to `Switch`. It prefers `props.location` over the internal router location, which enables the saved child element to be renered later and continue to match the old location.
 
 There are a handful of props handed to `AnimatedChild` that the parent will know about as it manages the animation. Again, this guide is more inspiration than copy/paste right now, feel free to look at the source of this website for exact implementation. Alright, let's check out the implementation of `AnimatedChild` (it's copy pasted from the animation used on this site).
 
 ```jsx
 class AnimatedChild extends Component {
-
   static propTypes = {
     children: PropTypes.node,
     anim: PropTypes.object,
     atParent: PropTypes.bool,
     animating: PropTypes.bool
-  }
+  };
 
   state = {
     // we're going to save the old children so we can render
     // it when it doesnt' actually match the location anymore
     previousChildren: null
-  }
+  };
 
-  componentWillReceiveProps(nextProps) {
+  componentDidUpdate(prevProps) {
     // figure out what to do with the children
-    const navigatingToParent = nextProps.atParent && !this.props.atParent
-    const animationEnded = this.props.animating && !nextProps.animating
+    const navigatingToParent = prevProps.atParent && !this.props.atParent;
+    const animationEnded = this.props.animating && !prevProps.animating;
```

Comment:
This one actually does need to be swapped around because it's checking if we were animating before and are no longer animating now.